There is a file descriptor leak in an errors path in AccountingProcess.cpp.
--- a/AccountingProcess.cpp
+++ b/AccountingProcess.cpp
@@ -478,12 +478,14 @@
 	string exe=string(context->conf.getVsaScript()) + " " + string(context->conf.getVsaNamedPipe());
 	if (write (fd_fifo, buf, buflen) != buflen)
 	{
+	  close(fd_fifo);
 	  cerr << getTime() << "RADIUS-PLUGIN: Could not write in Pipe to VSAScript!";
       return -1;
 	}
 	
 	if(system(exe.c_str())!=0)
 	{
+		close(fd_fifo);
 		cerr << getTime() << "RADIUS-PLUGIN: Error in VSAScript!";
 		return -1;
 	}
